css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,51 @@
+@import "../page-vars";
+
+.lune .ampstart-headerbar {
+  display: flex;
+  align-items: center;
+
+  height: var(--layout-header-height-m);
+
+  box-shadow: none;
+
+  will-change: transform;
+}
+
+.lune .ampstart-headerbar-title {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  font-size: 1.3125rem; /* 21px */
+  font-weight: var(--font-weight-black);
+
+  letter-spacing: .52em;
+  line-height: 1em;
+
+  /* Using same indent as the letter spacing to make the text properly centered. */
+  text-indent: .52em;
+
+  transform: translate(-50%, -50%);
+}
+
+.lune .ampstart-navbar-trigger {
+  padding-left: 0.3125rem; /* 5px */
+
+  font-size: 2.5rem; /* 40px */
+}
+
+@media (--breakpoint-mt) {
```

Comment:
We noticed that the gap between 640 and 830 was too large and needed a breakpoint in between. I would cater for an iPad in portrait mode.